package.json

```diff
@@ -44,12 +44,12 @@
     "tmp-promise": "3.0.2"
   },
   "dependencies": {
-    "@accordproject/concerto-analysis": "3.19.0",
-    "@accordproject/concerto-codegen": "3.23.4",
-    "@accordproject/concerto-core": "3.19.0",
-    "@accordproject/concerto-cto": "3.19.0",
-    "@accordproject/concerto-metamodel": "3.10.0",
-    "@accordproject/concerto-util": "3.19.0",
+    "@accordproject/concerto-analysis": "3.19.5",
+    "@accordproject/concerto-codegen": "3.30.0",
+    "@accordproject/concerto-core": "3.19.4",
```

Comment:
This is a first stab to update the deps, would bump to latest versions of concerto in a follow up